.github/workflows/periodic-build-new-actions-vm.yml

```diff
@@ -0,0 +1,50 @@
+name: Periodic Build New Actions VM
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.24.x'
+      - name: Install dependencies
+        run: cd ci/gha-runner-vm && go get .
+      - name: Install Packer
+        run: |
+          curl -LO https://releases.hashicorp.com/packer/1.12.0/packer_1.12.0_linux_amd64.zip
+          unzip packer_1.12.0_linux_amd64.zip
+          sudo mv packer /usr/local/bin/
+          rm packer_1.12.0_linux_amd64.zip
+          packer plugin install github.com/hashicorp/oracle
+      - name: Install OCI CLI
+        env:
+          OCI_CLI_USER: ${{ secrets.OCI_CLI_USER }}
+          OCI_CLI_TENANCY: ${{ secrets.OCI_CLI_TENANCY }}
+          OCI_CLI_FINGERPRINT: ${{ secrets.OCI_CLI_FINGERPRINT }}
+          OCI_CLI_KEY_CONTENT: ${{ secrets.OCI_CLI_KEY_CONTENT }}
+          OCI_CLI_REGION: ${{ secrets.OCI_CLI_REGION }}
+          OCI_INSTANCE_ID: ${{ secrets.OCI_INSTANCE_ID }}
+        run: |
+          curl -L -O https://raw.githubusercontent.com/oracle/oci-cli/master/scripts/install/install.sh
+          chmod +x install.sh
+          ./install.sh --accept-all-defaults
+          echo 'export PATH=$PATH:$HOME/bin' >> $GITHUB_ENV 
+          export PATH=$PATH:$HOME/bin
+          oci compute image list --compartment-id ocid1.compartment.oc1..aaaaaaaa22icap66vxktktubjlhf6oxvfhev6n7udgje2chahyrtq65ga63a --operating-system runner-images --operating-system-version 123456
+        
+      - name: Test with the Go CLI
+        env:
+          OCI_CLI_USER: ${{ secrets.OCI_CLI_USER }}
+          OCI_CLI_TENANCY: ${{ secrets.OCI_CLI_TENANCY }}
+          OCI_CLI_FINGERPRINT: ${{ secrets.OCI_CLI_FINGERPRINT }}
+          OCI_CLI_KEY_CONTENT: ${{ secrets.OCI_CLI_KEY_CONTENT }}
+          OCI_CLI_REGION: ${{ secrets.OCI_CLI_REGION }}
+          OCI_INSTANCE_ID: ${{ secrets.OCI_INSTANCE_ID }}
```

Comment:
These environment variables are defined twice; maybe we can put them under the job instead of the step.